docs/content/configuration-and-management/maas-controller-overview.md

```
@@ -1,346 +1,215 @@
-        HR["HTTPRoutes"]
-        AP2["maas-api AuthPolicy"]
-    end
+### Installation
 
-    subgraph "openshift-ingress namespace"
-        AP["gateway AuthPolicy"]
-        DR["DestinationRule"]
-        TP["TelemetryPolicy"]
-        IT["Istio Telemetry"]
-    end
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)